src/rma/destruction/templates/destruction/destructionlist_create.html

```diff
@@ -0,0 +1,25 @@
+{% extends "master.html" %}
+
+{% block content %}
+
+<article id="content-main">
+
+    {{ zaaktypen|json_script:"zaaktype-choices" }}
+    {{ reviewers|json_script:"reviewer-choices" }}
+
+    {{ form.errors }}
+    {{ form.non_field_errors }}
+
+        <div class="destruction-create"
+             id="react-destruction-list"
+             data-zaken-url="{% url 'destruction:fetch-zaken' %}"
+             data-url="{% url 'destruction:record-manager-create' %}"
+             data-csrftoken="{{ request.COOKIES.csrftoken }}"
```

Comment:
`csrf_token` as template variable is available through `django.template.context_processors.csrf`
```suggestion
             data-csrftoken="{{ csrf_token }}"
```